source/developers-guide/shopware-config/index.md

```diff
@@ -0,0 +1,139 @@
+---
+layout: default
+title: config.php settings
+github_link: developers-guide/shopware-config/index.md
+indexed: true
+group: Developer Guides
+subgroup: General Resources
+menu_title: config.php settings
+menu_order: 45
+---
+ 
+<div class="toc-list"></div>
+
+## Introduction
+In this guide we will take a closer look at the configuration file `config.php`.
+This file is in the root folder of a shopware installation. Normally it is generated during the
+installation process and filled with your database credentials. 
+It should look like this:
+```
+<?php
+return [
+    'db' => [
+        'username' => 'yourUsername',
+        'password' => 'yourPassword',
+        'dbname' => 'yourDbname',
+        'host' => 'yourHost',
+        'port' => 'yourPort'
+    ],
+];
+```
+During this guide I will point out some important options and explain them.  
```

Comment:
"I will point out" sounds really like a blog post we should write this more as a reference documentation